doc/ref_model/chapters/chapter03.md

```diff
@@ -124,24 +124,26 @@ _**Example**: a virtual compute descriptor as defined in TOSCA Simple Profile fo
 <a name="3.2.1.3"></a>
 #### 3.2.1.3 Virtual Storage
 
-A workload can request different types of storage based on data longevity: persistent or ephemeral storage.
+A workload can request different sizes and types of storage based on data longevity: persistent or ephemeral storage.
```

Comment:
typo: 'size' instead of 'sizes'

Is the 'size' determined by 'data longevity'?

```suggestion
A workload can request storage based on data longevity (persistent or ephemeral storage), different types of storage (HDD, SSD, etc.) and storage size.
```

Reply:
LGTM, thanks